newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -506,6 +522,22 @@ func expandDashboardPieWidgetConfigurationInput(i map[string]interface{}, meta i
 	}
 	return nil, nil
 }
+
+func expandDashboardLogTableWidgetConfigurationInput(i map[string]interface{}, meta interface{}) (*dashboards.DashboardLogTableWidgetConfigurationInput, error) {
+	var cfg dashboards.DashboardLogTableWidgetConfigurationInput
```

Comment:
This (and one or two other changes) require updates to `types.go` in a separate package, not sure where to go for that.

Reply:
Ah - seems to be https://github.com/newrelic/newrelic-client-go.